homeassistant/components/climate/spider.py

```diff
@@ -0,0 +1,151 @@
+"""
+Support for Spider thermostats.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/climate.spider/
+"""
+
+from datetime import timedelta
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.climate import (
+    ATTR_TEMPERATURE, PLATFORM_SCHEMA, STATE_COOL, STATE_HEAT, STATE_IDLE,
+    SUPPORT_TARGET_TEMPERATURE, ClimateDevice)
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME, TEMP_CELSIUS
+import homeassistant.helpers.config_validation as cv
+from homeassistant.util import Throttle
+
+REQUIREMENTS = ['itho_daalderop_api==1.0.3']
+
+_LOGGER = logging.getLogger(__name__)
+
+MIN_TIME_BETWEEN_UPDATES = timedelta(minutes=15)
```

Comment:
Is this throttling time the absolute min time to not be blocked by the api? If you just want to set the scan interval, define the `SCAN_INTERVAL` constant at the module level.

With this throttle time we won't get state feedback during service calls if they happen more often than every 15 min. That seems likely to happen, when changing some settings.

Reply:
I am not sure if throttling is needed. Trying currently without to see if the API doesn't kick me out.